considerations.md

```diff
@@ -0,0 +1,136 @@
+# Extensibility
+
+Implementations that are reading/processing manifests MUST NOT generate an error if they encounter an unknown property.
+Instead they MUST ignore unknown properties.
+
+# Canonicalization
+
+* OCI Artifacts are [content-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage). See [descriptors](descriptor.md) for more.
+* One benefit of content-addressable storage is easy deduplication.
+* Many artifacts might depend on a particular blob, but there will only be one blob in the store.
+* With a different serialization, that same semantic blob would have a different hash, and if both versions of the blob are referenced there will be two blobs with the same semantic content.
+* To allow efficient storage, implementations serializing content for blobs SHOULD use a canonical serialization.
+* This increases the chance that different implementations can push the same semantic content to the store without creating redundant blobs.
+
+## JSON
+
+[JSON][] content SHOULD be serialized as [canonical JSON][canonical-json].
```

Comment:
I find it hard to recommend canonical json.

Reply:
We can change this, but it was simply [copied from the image-spec](https://github.com/opencontainers/image-spec/blob/master/considerations.md)